salt/pillar/http_yaml.py

```diff
@@ -66,8 +66,9 @@ def __virtual__():
 def ext_pillar(minion_id,
                pillar,  # pylint: disable=W0613
                url,
-               with_grains=False,
-               **kwargs):
+               username=None,
+               password=None,
```

Comment:
One last thing, we have to maintain backwards compatibility, and if someone is passing in a True or False in to with_grains without the keyword, then this will break it.

So any new keyword arguments do need to be added to the end of the list.

Thanks!
Daniel

Reply:
Okey I did not think of that. Thanks for your quick reply and your patience.

Armin